bindings/pyroot_experimental/pythonizations/python/ROOT/_facade.py

```diff
@@ -81,13 +81,29 @@ def __init__(self, module, is_ipython):
         # Setup import hook
         self._set_import_hook()
 
-    def AddressOf(self, *args):
-        # Return a bytearray that can fit a long long, which is what addressof
-        # returns (wrapped in a Python integer). The bytes of the bytearray
-        # correspond to the address of the object in args
-        import struct
-        ad = self.addressof(*args)
-        return bytearray(struct.pack('q', ad))
+    def AddressOf(self, obj):
+        # Return an indexable buffer of length 1, whose only element
+        # is the address of the object.
+        # The address of the buffer is the same as the address of the
+        # address of the object
+
+        # addr is the address of the address of the object
+        addr = self.addressof(instance = obj, byref = True)
+
+        # get_llv returns a buffer (LowLevelView)
+        try:
+            get_llv = gbl_namespace.PyROOT.get_llv
+        except AttributeError:
+            gbl_namespace.gInterpreter.Declare("""
```

Comment:
You could put this into `libROOTPythonization.so`, no?

Reply:
Theoretically, yes. I did it with this jitted function (which is jitted only on first invocation of `AddressOf`), because the fact of returning a `long long*` automatically makes cppyy construct for me a `LowLevelView` object.

I can try to do this in `libROOTPythonization.so`, but I need to find how to construct that `LowLevelView` object myself. Will try and if I succeed I will modify this as you suggested.